src/index.html

```diff
@@ -6,7 +6,440 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header">
+      <a href="#home" class="page__headerLogo">
+        <img
+          src="images/logo.png"
+          alt="Logo imagine"
+          class="img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
```

Comment:
let's stick to one BEM naming convention
```suggestion
    <header class="page__header">
      <a href="#home" class="page__headerLogo">
        <img
          src="images/logo.png"
          alt="Logo imagine"
          class="img"
        >
      </a>
      <nav class="nav">
        <ul class="nav__list">
```